flowetl/Dockerfile

```diff
@@ -6,7 +6,7 @@
 #  FLOWETL
 #  -----
 
-FROM puckel/docker-airflow
+FROM greenape/docker-airflow
```

Comment:
Again out of interest, what are the tweaks in this image compared to the puckel one?

Reply:
It uses the newest release of airflow... Not sure if should pull that into the repo in its entirety or leave it where it is?

Reply:
Might want to take a look at some of the open PRs on the puckle repo before switching - I rolled this as a poc for airflow now working with python 3.7

Reply:
There's a PR but been there a while... There is a docker file in the official airflow repo perhaps should make use of that?

Reply:
puckel/docker-airflow/pull/358

Reply:
https://github.com/apache/airflow/blob/master/Dockerfile

Reply:
The airflow official one - man, why even put it out there if you're going to start it by saying don't use it? 🙄 

https://github.com/puckel/docker-airflow/pull/356 seems to include a _bunch_ of additional fixes, which is why I'm wondering if sticking with @puckel's image might not be better? For the 3.7 thing, I wonder if we'd be better off submitting a PR over there which updates the python image/adds a 3.7 tag?

Reply:
Opened https://github.com/puckel/docker-airflow/issues/364 re: the python version.